ti/testsplitter/consts.go

```diff
@@ -0,0 +1,14 @@
+package testsplitter
+
```

Comment:
can you use test_splitter as the directory name?

Reply:
Not encouraged in Go. Hence had to shift to `testsplitter`.